README.md

```diff
@@ -7,3 +7,17 @@ Use [this app](https://grokdebug.herokuapp.com/) for debugging your
 patterns! Be careful, there might be subtle differences!
 
 When issuing pull requests to this repository, don't forget to include an example of the messages your commits try to parse!!
+
+
+# Adding pattern
+
+Develop a new pattern typically in itis own file, and call the main new one `<something>_MSG`.
+
+Than you can do:
+ * extend the `RSYSLOGMESSAGE` in the `rsyslog` file with a new pattern,
+   by joining it with a `|` and placing the new pattern before the `GREEDYDATA` one.
+   This requires only a new rpm and no configuration changes.
+ * add the new pattern to the list in the test configuration to the `grok`
+   filter before the `RSYSLOGMESSAGE` as follows: `%{RSYSLOGPREFIX}%{<something>_MSG}`
+   This requires a new rpm and configuration change in quattor too.
```

Comment:
Given that you moved all the existing ones to the grok filter, why should this not be the default place to add stuff? Or should this only be used once it's stable, given that it requires two changes?